lib/fake_data_loader.rb

```diff
@@ -37,4 +42,37 @@ def self.load_admin
     admin = AdminUser.find_by(email: email)
     AdminUser.create!(email: email, password: pass, password_confirmation: pass) unless admin
   end
+
+  def self.load_users
+    10.times do
+      user = create(
+        :user,
+        email: Faker::Internet.email,
+        password: USER_PASSWORD
+      )
+
+      puts "user: #{user.email} - password: #{USER_PASSWORD}"
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Rails/Output: Do not write to stdout. Use Rails's logger if you want to log.